src/components/Button.js

```diff
@@ -135,6 +137,19 @@ class Button extends Component {
         this.unsubscribe();
     }
 
+    /**
+     * Setup and attach keypress handler for pressing the button with Enter key
+     */
+    subscribeEnterKey() {
+        const shortcutConfig = CONST.KEYBOARD_SHORTCUTS.ENTER;
+        this.unsubscribe = KeyboardShortcut.subscribe(shortcutConfig.shortcutKey, () => {
+            if (this.props.isDisabled || this.props.isLoading) {
+                return;
+            }
+            this.props.onPress();
+        }, shortcutConfig.descriptionKey, shortcutConfig.modifiers, this.props.pressOnEnter);
```

Comment:
This issue only affects Web-based platforms. (mWeb | Web | Desktop). 
Thus we can simply rely on and give up all other logic.

```suggestion
        this.unsubscribe = KeyboardShortcut.subscribe(shortcutConfig.shortcutKey, (e) => {
            if (this.props.isDisabled || this.props.isLoading || !e || e.target.nodeName === 'textarea' ) {
                return;
            }
            this.props.onPress();
        }, shortcutConfig.descriptionKey, shortcutConfig.modifiers, this.props.pressOnEnter);
```

Reply:
but we can do it from a keyboard subscription file check the change i make now @parasharrajat

Reply:
I see what you mean and why that is happening. Let's hold this PR. Soon we should be able to do this. Till then, we can wait.

Reply:
ok, but the current change I do work fine . 
what do you think about it

Reply:
I don't like it it seems unnecessarily complicated.

Reply:
@parasharrajat  no, I discard all this and just check on the node on the keyboard shortcut 
check here again. 
https://github.com/Expensify/App/pull/7920/files

Reply:
@parasharrajat can you clarify this comment please? https://github.com/Expensify/App/pull/7920#discussion_r815302926

Reply:
I am waiting for event bubbling to be added to the KeyboardShortcut handler so that our solution can work without hacking around. https://github.com/Expensify/App/pull/7702